content/docs/platform/quickstart/nextjs.mdx

```diff
@@ -66,75 +55,74 @@ This guide walks you through integrating Novu’s Inbox into your Next.js applic
   </Step>
 
   <Step>
-  ### Create an Inbox component
-    In the `src` directory of your Next.js project, create a `components/inbox.tsx` file with the following content:
+  ### Add the notification Inbox to your app
+    Import Novu’s built-in <Method href="/platform/inbox/react/components/inbox">{`<Inbox />`}</Method> component into your layout file and place it in the navbar:
 
-    <InboxCodeBlock />
+    ```tsx title="app/layout.tsx"
+      import { Inbox } from '@novu/nextjs'; // [!code highlight]
+      import './globals.css';
+
+      export const metadata = {
+        title: 'Novu Next.js Quickstart',
+        description: 'Generated by create next app',
+      };
+
+      export default function RootLayout({ children }) {
+        return (
+          <html lang="en">
+            <body>
+            // [!code highlight]
+              <nav className="flex justify-start items-center p-4 gap-4 h-16"> 
+                <Inbox // [!code highlight]
+                  applicationIdentifier="YOUR_APPLICATION_IDENTIFIER" // [!code highlight]
+                  subscriberId="YOUR_SUBSCRIBER_ID" // [!code highlight]
+                  // [!code highlight]
```

Comment:
WE should use the previous inbox code block component, as it auto injects the Keys here for logged in users